### PR TITLE
Docs pick 6.2

### PR DESCRIPTION
**Pull request by randyh62**

### CODE REVIEW

File: docs/reference/types.rst [hunk 1]
```diff
@@ -10,8 +10,8 @@ rocSOLVER Types
 
 rocSOLVER uses most types and enumerations defined in rocBLAS for the general operation and
```
Comment (lpaoletti) [inline docs/reference/types.rst:11]: Perhaps we should get away from the "additional types" terminology - this is not what the topic title indicates:

image

Perhaps add this paragraph after the intro paragraph:
This topic describes the rocSOLVER types that extend the rocBLAS and rocSPARSE APIs.

File: docs/index.rst [hunk 2]
```diff
@@ -8,12 +8,12 @@
 rocSOLVER documentation
 ********************************************************************
 
-rocSOLVER is an implementation of `LAPACK routines <https://www.netlib.org/lapack/explore-html/modules.html>`_
-on top of `AMD's open source ROCm platform <https://rocm.docs.amd.com/>`_. rocSOLVER is implemented in the
-`HIP programming language <https://rocm.docs.amd.com/projects/HIP/>`_ and optimized for AMD's
+rocSOLVER is an implementation of `LAPACK routines <https://www.netlib.org/lapack/explore-html/modules.html>`_ 
+on top of :doc:`AMD's open source ROCm platform <rocm:index>`. rocSOLVER is implemented in the
```
Comment (lpaoletti) [inline docs/index.rst:12]:
```suggestion
on top of the :doc:`AMD ROCm platform <rocm:index>`. rocSOLVER is implemented in the
```

File: docs/index.rst [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
+:doc:`HIP programming language <hip:index>` and optimized for AMD's
 latest discrete GPUs.
```
Comment (lpaoletti) [inline docs/index.rst:14]:
```suggestion
:doc:`HIP programming language <hip:index>` and optimized for AMD GPUs.
```

File: docs/installation/installlinux.rst [hunk 4]
```diff
@@ -43,7 +43,7 @@ rocSOLVER can be installed using a package manager. On Ubuntu, for example, use
 Building & installing from source
```
Comment (lpaoletti) [inline docs/installation/installlinux.rst:43]:
```suggestion
Building and installing from source
```

File: docs/installation/installlinux.rst [hunk 5]
```diff
@@ -12,11 +12,11 @@ Prerequisites
 =================
```
Comment (lpaoletti) [inline docs/installation/installlinux.rst:12]: Sentence case for the title above would be good

**Review by lpaoletti — COMMENTED** [review]
This topic is not needed:
image

**Comment by yhuiYH** [comment]
@lpaoletti This is a cherry-pick task, there should be no content changes.  Content changes should target DEVELOP branch.  
The goal is to leave the content, as is, and get it into release/rocm-rel-6.2 branch.  I think this PR looks fine, for what it was intended for :).

**Review by samjwu — APPROVED** [review]

**Review by yhuiYH — APPROVED** [review]
LGTM for cherry pick into release/rocm-rel-6.2

**Pull request merged**